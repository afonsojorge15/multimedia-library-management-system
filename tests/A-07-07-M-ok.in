1
naoExistente
0
