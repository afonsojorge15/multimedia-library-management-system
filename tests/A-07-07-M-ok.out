Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Ficheiro a abrir: Abrir dados: Operação inválida: O ficheiro 'naoExistente' não existe.
Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 